addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2011,6 +2011,7 @@ var FieldBinaryImage = AbstractFieldBinary.extend({
     },
     _render: function () {
         var self = this;
+        this.placeholder = this.attrs.placeholder || this.placeholder;
```

Comment:
Can you avoid using the placeholder attribute and remove this?

As I suggested earlier, I still think there is still merit to calling `/web/image` and letting `_get_placeholder_filename` do the job.